src/databricks/labs/ucx/install.py

```diff
@@ -197,8 +197,11 @@ def _is_testing(self):
 
     def _prompt_for_new_installation(self) -> WorkspaceConfig:
         logger.info("Please answer a couple of questions to configure Unity Catalog migration")
+        default_database = "ucx"
+        if self.policy_installer.has_ext_hms():
+            default_database = f"ucx_{self.workspace_client.get_workspace_id()}"
```

Comment:
Why do we want to postfix the `_<workspace id>` when there is a single hive metastore?

Reply:
if a workspace is configured to use external hive metastore, majority of the time that metastore will be shared with other workspaces. we need to add the suffix to ensure uniqueness of the inventory database

Reply:
Oke, thanks, please add that as a comment.

Reply:
done :)